maas/resource_maas_network_interface_bridge.go

```diff
@@ -0,0 +1,272 @@
+					"mtu":         n.EffectiveMTU,
+					"bridge_fd":   n.BridgeFD,
+					"bridge_stp":  n.BridgeSTP,
+					"bridge_type": n.Params.(map[string]interface{})["bridge_type"],
+					"tags":        n.Tags,
+				}
+				if err := setTerraformState(d, tfState); err != nil {
+					return nil, err
+				}
+				return []*schema.ResourceData{d}, nil
+			},
+		},
+
+		Schema: map[string]*schema.Schema{
+			"machine": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The identifier (system ID, hostname, or FQDN) of the machine with the bridge network interface.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Optional:    false,
+				Computed:    false,
+				Description: "The name of the the bridge network interface.",
+			},
+			"vlan": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "MAAS VLAN ID the interface is connected to.",
+			},
+			"parent": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: "Parent interface ID for this bridge interface.",
+			},
+			"mtu": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "The MTU of the bridge network interface. This argument is computed if it's not set.",
+			},
+			"bridge_type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    false,
+				Description: "The type of bridge to create. Possible values are: standard, ovs.",
+				Default:     "standard",
+			},
+			"bridge_fd": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Set bridge forward delay to time seconds. (Default: 15).",
+			},
+			"bridge_stp": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Turn spanning tree protocol on or off. (Default: False).",
+			},
+			"mac_address": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "MAC address of the interface",
+			},
+			"tags": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				Description: "Tags for the interface.",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+		},
+	}
+}
+
+func resourceNetworkInterfaceBridgeCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	client := m.(*client.Client)
+
+	machine, err := getMachine(client, d.Get("machine").(string))
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	networkInterface, err := findNetworkInterfaceBridge(client, machine.SystemID, d.Get("name").(string))
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	if networkInterface == nil {
+		networkInterface, err = client.NetworkInterfaces.CreateBridge(machine.SystemID, getNetworkInterfaceBridgeParams(d))
+		if err != nil {
+			return diag.FromErr(err)
+		}
+	}
+	d.SetId(fmt.Sprintf("%v", networkInterface.ID))
```

Comment:
`networkInterface.ID` should always be an int, can you change the `fmt.Sprintf()` to `strconv.Itoa()`